src/user/managing-dashboards.md

```diff
@@ -296,84 +259,64 @@ To view the chart, table or map in fullscreen, click on the **View fullscreen**
 
-All dashboards have two sharing groups set by default.
+There are 3 levels of sharing permissions available for a dashboard:
 
-- External access (without login)
+- No access
 
-  This option, when selected, provides access to the dashboard as an
-  external resource through the API. This is useful for when you are creating an
-  external web portal but would like to call information from a
-  dashboard you have made internally within DHIS2. By default, this
-  option is not selected. For more information, see [Viewing analytical resource representations](https://docs.dhis2.org/master/en/developer/html/webapi_viewing_analytical_resource_representations.html#) in the developer guide.
+  The user or user group cannot view or edit the dashboard. If they try to access the dashboard with the url, the Dashboards app will display the message "Requested dashboard not found".
 
-- Public access (with login)
+- View only
 
-  This option allows the selected dashboard to be pushed to all users
-  within your DHIS2 instance. This can also be hidden from public view
-  by selecting the "None" option, which is the default option for new
-  dashboards.
+  The user or user group can view the dashboard but cannot edit it.
 
-User groups that have been added manually can be assigned two types of
-permissions within the dashboard
+- View and edit
+
+  The user or user group can view and edit the dashboard. Editing allows for altering the layout, resizing and removing items, renaming and deleting the dashboard, etc.
+
+All dashboards have two sharing groups set by default.
 
-- Can view
+- External users
 
-  Provides the user group with view only rights to the dashboard.
+  The *External users* group defines access to the dashboard for people not logged in to the DHIS2 instance. By default the setting is **No access** and can be changed to **View only**. Giving access to external users is useful for when you are creating an external web portal and would like to present information from DHIS2.
 
-- Can edit and view
+- All users
 
-  Allows the user groups to edit the dashboard in addition to viewing
-  it. Editing allows for altering the layout, resizing and removing
-  items, renaming/deleting the dashboard etc.
+  The *All users* group defines access to the dashboard for all logged-in users in the DHIS2 instance. By default the setting is **No access** and can be changed to **View only** or **View and edit**.
 
-You can provide users with the url of the dashboard, allowing them to
-navigate directly to the dashboard. To get the dashboard url, just
-access the dashboard in view mode, and copy the browser url. For
-example, the url to the Antenatal Care dashboard in play.dhis2.org/demo
-is:
+### Add users and user groups
+
+To share a dashboard with specific users and user groups, type the name in the input field, choose the desired access level and click on **Give access**.
+
+![](resources/images/dashboard/dashboard-sharing-add-user.png)
```

Comment:
oh how the OCD alert goes off when the first screenshot has a dashboard named "test" and the second one is named "Test" 
![image](https://user-images.githubusercontent.com/12590483/134308949-fd40ff51-748e-4180-8ea8-a7f037bd1aef.png)
![image](https://user-images.githubusercontent.com/12590483/134309008-634016a7-67eb-4bee-8cde-568b7539dadf.png)